R/build-articles.R

```diff
@@ -102,36 +105,63 @@ render_rmd <- function(pkg,
                        toc = TRUE,
                        depth = 1L,
                        encoding = "UTF-8",
+                       render_asis = FALSE,
                        quiet = TRUE) {
 
   message("Building article '", output_file, "'")
   scoped_package_context(pkg$package, pkg$topic_index, pkg$article_index)
   scoped_file_context(depth = depth)
 
-  format <- build_rmarkdown_format(pkg, depth = depth, data = data, toc = toc)
-  on.exit(unlink(format$path), add = TRUE)
+  if (!render_asis)
```

Comment:
Could you please take a look at http://style.tidyverse.org?

I think the main changes needed are to put the `{` on the same line at the `if`, and to do assignment inside the individual branches, rather than outside the if statement.